src/main/java/com/school/project/model/entities/Lesson.java

```diff
@@ -1,14 +1,42 @@
 package com.school.project.model.entities;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
 import java.util.Date;
+@Entity
+@Table(name = "LESSON")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
 
 public class Lesson {
 
+    @Id
+    @Column(name = "LESSON_ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
+    @Column(name="THEMA", nullable = false)
```

Comment:
Pls use it: The auto-formatting shortcut in IntelliJ

For windows Ctrl+Alt+L.
For ubuntu Ctrl+Alt+windows+L.
For Mac Command+Option+L.